packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -116,9 +116,11 @@ class BottomSheet extends StatefulWidget {
   /// Default is true.
   final bool enableDrag;
 
-  /// Specifies whether drag handler is displayed when the drag action is enabled.
+  /// Specifies whether a drag handle is shown when [enableDrag] is true.
   ///
-  /// Defaults is true in material 3 and false in material 2.
+  /// Defaults to [BottomSheetThemeData.hasDragHandle], if it is also null, 
+  /// behaves as true when [ThemeData.useMaterial3] is true, and otherwise 
+  /// behaves as false.
```

Comment:
nit to clarify the grammar (make more explicit what "it" refers to):
```suggestion
  /// Defaults to [BottomSheetThemeData.hasDragHandle]. If that is also null,
  /// behaves as true when [ThemeData.useMaterial3] is true, and otherwise
  /// behaves as false.
```

Similarly on the ModalBottomSheetRoute field.